requirements.txt

```diff
@@ -1,8 +1,8 @@
 PyYAML==3.11
 cached-property==1.2.0
-dockerpty==0.3.4
 docopt==0.6.1
 enum34==1.0.4
+git+https://github.com/d11wtq/dockerpty.git@master#egg=dockerpty
```

Comment:
I think we should be using a sha here, otherwise it could break without any changes to compose